docs/source/workbench/superplot.rst

```diff
@@ -0,0 +1,41 @@
+.. _WorkbenchSuperplot:
+
+=========
+Superplot
+=========
+
+Superplot is a decorator widget of the plot window. It facilitates over-plotting
+and manipulation of overplotted data. The superplot is accessible:
+
+* Directly in the plot window through the menu bar
```

Comment:
It's not the menu bar, it's the toolbar.

Reply:
done